src/components/html/EcAppCard.html

```diff
@@ -1,25 +1,11 @@
-<a-card class="ec-app-card" @click="$emit('click')" hoverable>
-  <div slot="cover" class="text-center">
-    <img
-      v-if="iconUrl"
-      class="img-fluid"
-      :src="iconUrl"
-      :alt="app.title"
-    />
-    <a-icon v-else type="picture"/>
+<div class="card" @click="$emit('click')" hoverable style="height: 26rem">
+  <img v-if="iconUrl" class="card-img-top" :src="iconUrl" :alt="app.title" />
+  <div class="card-body">
+    <h5 style="white-space: nowrap; width: 100%; overflow: hidden; text-overflow: ellipsis;" class="card-text">
+      {{app.title}}</h5>
+    <h6 class="card-subtitle mb-2 text-muted" style="margin-top: 20px;">{{ formattedDescription }} </h6>
   </div>
-
-  <a-card-meta
-    :title="app.title"
-    class="overflow-hidden"
-    style="height: 9.25rem"
-  >
-    <template slot="description">
-      {{ formattedDescription }}
-    </template>
-  </a-card-meta>
-
-  <span class="text-success" v-if="!app.paid">
+  <span class="text-success" style="font-size: 16px; padding: 20px;" v-if="!app.paid">
```

Comment:
Por quê este padding e font-size mockado aqui? Como te falei, não é a ideia manter idêntico ao que tava antes, em geral não precisa (nem deve) forçar tamanho de fonte e espaçamento (piorou com inline style), isto vai ser definido pelos estilos dos componentes do Bootstrap mesmo, com o nosso tema...

Reply:
Aqui também se aplica https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended